src/content/protocols/sky.md

```diff
@@ -0,0 +1,387 @@
+---
+protocol: "Sky"
+website: "https://sky.money/"
+x: "https://x.com/SkyEcosystem"
+github: ["https://github.com/makerdao"]
+defillama_slug: ["sky-lending"]
+chain: "Ethereum"
+stage: 0
+reasons: []
+risks: ["L", "H", "M", "H", "L"]
+author: ["mmilien_"]
+submission_date: "2025-03-11"
+publish_date: "1970-01-01"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+The Sky protocol is a decentralised protocol developed around the `USDS` stablecoin. It is managed by the Sky ecosystem governance. The Sky Protocol features Sky tokens (`USDS`, `SKY`, `DAI`, `MKR`), the Sky Savings Rate, Sky Token Rewards, Activation Token Rewards, and SkyLink. The Protocol was built from the core module of the MakerDAO Protocol and replaces Maker. As Sky is built on top of Maker and both protocol's tokens are tied by fixed conversion rate, this review also includes the MakerDAO protocol and its contracts.
```

Comment:
Sky is a stablecoin protocol allowing users to mint its `USDS` stablecoin through _Collateralized Debt Positions_ with a variety of crypto collateral assets. Sky is built on the Maker protocol, by the MakerDAO, with the objective to replace the `DAI` stablecoin and allows for the 1:1 conversion between DAI and USDS. USDS can further be staked for the yield-bearing `sUSDS` token. Various levels of control over the Sky protocol are exercised through an onchain governance system.